recipe/meta.yaml

```
@@ -1,17 +1,17 @@
 {% set name = "mamonca" %}
-{% set version = "0.0.5" %}
+{% set version = "0.0.7" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/mamonca-{{ version }}.tar.gz
-  sha256: e01df7aeb53de31e701b9459bdfb3676bfe90cae1967f521371f211809959129
+  sha256: c1c928a3542ffb65c233b517d4c4e9e84683f27b0548353792c594cb0cc4ea9d
 
 build:
   script: {{ PYTHON }} -m pip install . -vv
-  number: 2
+  number: 0
```

As there is already a release for `build: 0` I would recommend to set `build: 1` here:
```suggestion
  number: 1
```